tests/lit-tests/debug_type_dwarf.ispc

```diff
@@ -0,0 +1,12 @@
+// RUN: %{ispc} --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s --implicit-check-not "llvm.dbg.cu"
+// RUN: %{ispc} -g --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_CODEVIEW
+// RUN: %{ispc} -g --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_CODEVIEW
```

Comment:
The test is supposed to be run on Windows host only, right? Please use `WINDOWS_HOST` in REQUIRES labels.

Reply:
The test is supposed to be run on any host that supports windows as target-os. I thought that `WINDOWS_ENABLED` means exactly that.

Reply:
Oh, yeah. I have fixed that.

Reply:
Now line 2 and 3 are the same :) please remove one of them